From fd799756682b360cf879cb573596df18d2a31e0a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 02 Apr 2020 00:12:52 +0300
Subject: [PATCH] [SW-11180] preboot: rename *_image_offset()

Rename stable_image_offset() and modifiable_image_offset() functions to
stable_image_base_offset() and modifiable_image_base_offset()
accordingly.
The new names better reflect what these functions do.

Change-Id: I69acd5f9569f524a918ae9d9a92455ec28ea4084
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/preboot.c b/app/preboot.c
index 1da59bc..dc88745 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -25,7 +25,7 @@
 	" (" PREBOOT_DATE " - " PREBOOT_TIME ")"
 
 /* Stable ID can be 0 or 1, if not defined return primary offset (0) */
-static u32 stable_image_offset(void)
+static u32 stable_image_base_offset(void)
 {
 	u32 stable_image_id;
 
@@ -39,7 +39,7 @@
 	return HL_SPI_PRIMARY_BASE;
 }
 
-static u32 modifiable_image_offset(bool *pending)
+static u32 modifiable_image_base_offset(bool *pending)
 {
 	u32 ret = HL_SPI_PRIMARY_BASE;
 	u32 modifiable_id;
@@ -71,10 +71,10 @@
 
 	/* Check for normal boot */
 	if (image_pending != FLOPS_UPD_PENDING)
-		return stable_image_offset();
+		return stable_image_base_offset();
 
 	/* There is an update pending */
-	return modifiable_image_offset(pending);
+	return modifiable_image_base_offset(pending);
 }
 
 static void preboot_copy_uboot(void)
